frontend/src/app/pages/exercises/exercise/shared/trainer-map-editor/trainer-map-editor.component.html

```diff
@@ -31,119 +31,69 @@ <h5 class="mb-0 d-flex">
-                                <h5
-                                    class="card-title text-truncate"
-                                    [title]="'Ansicht'"
-                                >
-                                    Ansicht
-                                </h5>
-                            </div>
-                        </div>
-                        <div
-                            class="d-inline-block card text-center"
-                            style="width: 10rem; text-overflow: ellipsis"
-                            (mousedown)="
-                                dragElementService.onMouseDown($event, {
-                                    type: 'transferPoint',
-                                    template: transferPointTemplate,
-                                })
-                            "
-                            data-cy="draggableTransferPointDiv"
-                        >
-                            <div class="preview-image-container">
-                                <img
-                                    [src]="transferPointTemplate.image.url"
-                                    class="card-img-top"
-                                />
-                            </div>
-                            <div class="card-body">
-                                <h5
-                                    class="card-title text-truncate"
-                                    [title]="'Transferpunkt'"
-                                >
-                                    Transferpunkt
-                                </h5>
-                            </div>
-                        </div>
-                        <div class="p-2">
-                            <div class="form-check form-switch">
-                                <input
-                                    class="form-check-input"
-                                    type="checkbox"
-                                    [ngModel]="
-                                        transferLinesService.displayTransferLines
-                                    "
-                                    (ngModelChange)="
-                                        changeDisplayTransferLines($event)
-                                    "
-                                />
-                                <label class="form-check-label"
-                                    >Verbindungen Anzeigen</label
-                                >
-                            </div>
-                        </div>
+                            (ngModelChange)="changeDisplayTransferLines($event)"
+                        />
+                        <label class="form-check-label">
+                            Verbindungen anzeigen
+                        </label>
                     </div>
+                    <h5>Ansicht/Transferpunkt</h5>
+                    <app-map-editor-card
+                        [title]="'Ansicht'"
```

Comment:
For specifying literal values, you can simplify this to

```suggestion
                        title="Ansicht"
```

Also applies to `dataCy` and to both attributes on all other editor cards that use literal values